tests/oss-fuzz/build.sh

```diff
@@ -41,6 +41,7 @@ ln -s $SRC/fuzztest $SRC/libavif/ext/fuzztest
 # build dependencies
 cd ext && bash aom.cmd && bash dav1d.cmd && bash googletest.cmd && bash libjpeg.cmd && \
       bash libsharpyuv.cmd && bash libyuv.cmd && bash zlibpng.cmd && cd ..
+export CXXFLAGS="${CXXFLAGS} -DFUZZTEST_COMPATIBILITY_MODE"
```

Comment:
Just curious: This `FUZZTEST_COMPATIBILITY_MODE` doesn't need to be defined as "libfuzzer"?

Reply:
Not here no, this is just to compile some code. This is what is done with bazel is oss-fuzz now: https://github.com/google/fuzztest/blob/9c6d264c9bd77c2dcff17424e7e40ccb49f8e09d/bazel/setup_configs.sh#L116
The CMake counterpart does need to be defined as libfuzzer though (it is the only one that is supported for now)